web/src/server_events_dispatch.js

```diff
@@ -670,10 +668,10 @@ export function dispatch_normal_event(event) {
                                 "zulip_update_announcements_stream_id",
                             );
                         }
-                        if (is_narrowed_to_stream) {
-                            assert(message_lists.current !== undefined);
-                            message_lists.current.update_trailing_bookend(true);
```

Comment:
Removed this since rerendering the view removes the bookend and we anyways did not show bookend after reloading.